src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+// import '@testing-library/jest-dom';
```

Comment:
- [ ] Kindly delete this file since you are not using the codes in it. Removing all unused codes from your codebase makes your work look clean and ensures that you adhere to the _**YAGNI**_ principle. Kindly delete this file. 👍 😎